plugins/package-managers/python/src/funTest/assets/projects/synthetic/poetry-expected-output.yml

```diff
@@ -22,7 +22,210 @@ project:
     - id: "PyPI::jinja2:3.0.3"
       dependencies:
       - id: "PyPI::markupsafe:2.0.1"
+  - name: "installDev"
```

Comment:
Commit message:
- "not" -> "note"

Reply:
Commit message: The last paragraph seem to just repeat the very first sentence on different word, and could IMO be dropped.